Library/Homebrew/resource_auditor.rb

```diff
@@ -101,7 +101,7 @@ def audit_urls
 
         strategy = DownloadStrategyDetector.detect(url, using)
         if strategy <= CurlDownloadStrategy && !url.start_with?("file")
-          if (http_content_problem = curl_check_http_content(url, specs: specs))
+          if (http_content_problem = curl_check_http_content(url, "zip, tarball, or repo URL", specs: specs))
```

Comment:
```suggestion
          if (http_content_problem = curl_check_http_content(url, "source URL", specs: specs))
```
This seems nicer?

Reply:
Yes, thank you. I thought there was a more concise way to say it, but it was escaping my brain 😛 

Change made.